strawberry/types/graphql.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import enum
+from typing import Tuple
+
+
+class OperationType(enum.Enum):
+    QUERY = "query"
+    MUTATION = "mutation"
+    SUBSCRIPTION = "subscription"
```

Comment:
Are these values necessary? Can we use `auto()`?

Reply:
what's the benefit in using auto?

Reply:
It indicates that the enum is only ever used as a symbol and never for its value. It can be pretty useful when exploring an API

Reply:
we do use it by value: `OperationType(definition.operation.value)` inside https://github.com/strawberry-graphql/strawberry/blob/867e3a4b0d60a79262264cda43f060f195352c5b/strawberry/utils/operation.py#L35

Reply:
I'm a but confused why that's not just `return definition.operation`?

Reply:
that's a different enum (GraphQL-core's)

Reply:
Oh because that's graphql-core's operation :facepalm: